frontend/src/metabase/lib/urls/questions.ts

```diff
@@ -107,9 +107,10 @@ export function newQuestion({
   });
 
   const entity = question.isDataset() ? "model" : "question";
+  const type = question.type() ?? entity;
 
   if (mode) {
-    return url.replace(/^\/(question|model)/, `/${entity}\/${mode}`);
+    return url.replace(/^\/(question|model)/, `/${type}\/${mode}`);
```

Comment:
I'm relying on the existing behavior, and the fact that the metric is always initialized in a notebook mode.
There's weird and convoluted function execution order involved here.

We could be dealing with this logic inside the underlying `question` function
https://github.com/metabase/metabase/blob/37352-new-metric/frontend/src/metabase/lib/urls/questions.ts#L27

So why aren't we?
With the presumption that there was a good reason for it, I'll solve the logic here. But I'll make sure to follow up, and to investigate this call stack in the following PR.

cc @kamilmielnik